packages/core/package.json

```
@@ -15,6 +15,7 @@
   "dependencies": {
     "@apollo/client": "^3.7.17",
     "@ember/string": "^3.1.1",
+    "@embroider/macros": "1.13.0",
```

```suggestion
    "@embroider/macros": "^1.13.0",
```

This is already used in the distribution package and should be the same

---

done